### PR TITLE
Update descriptions for veto mediation

### PR DESCRIPTION
**Pull request by NoffyNoffeh**
Updating written documentation to reflect what was discussed during the public [NAT Meeting (summary doc)](https://docs.google.com/document/d/1G1DqWpzH-aVHQ1axqqVcHdzdNnNIa0WDcnkXGoM_SaU/edit?usp=sharing) that was held in the dev server July 14.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/People/Beatmap_Nominators/Beatmap_Veto/en.md [hunk 1]
```diff
@@ -26,10 +26,14 @@ If discussion is no longer progressing in the direction of a compromise, or is o
 
 If the vetoer is no longer a Beatmap Nominator and no mediation has been requested, the veto will be dismissed.
 
-During mediation, the Beatmap Nominators and members of the NAT will vote for whether to uphold or dismiss the veto anonymously. Once initiated, this process can take anywhere from a day up to a week depending on how quickly participants make their decisions.
+### Veto Mediation
```
Comment (Walavouchey) [inline wiki/People/Beatmap_Nominators/Beatmap_Veto/en.md:29]: headings are sentence case

```suggestion
### Veto mediation
```

File: wiki/People/Beatmap_Nominators/General_Information/en.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ In addition to being able to nominate beatmaps or reset their nominations, BNs h
 
 ### Veto mediation
 
-When a veto mediation is initiated, a randomly selected jury of BNs will be selected. If you are one of the selected members for a veto mediation, the NAT will send you a message with additional information on how to vote using the [BN Website](http://bn.mappersguild.com). If you are unable to participate, inform a NAT member so that a different BN can be chosen for the jury in your place.
+When a veto is sent to mediation, BNs and NAT members of the vetoed beatmap's mode will all be asked to submit individual votes on the issue. In the case of vetoes related to issues that are not mode-specific, all mode members will be asked to participate. NAT will send you a message with additional information on how to vote using the [BN Website](http://bn.mappersguild.com) when a veto mediation begins. At the end of mediation, all votes submitted will be compiled to determine a final upheld or dismissed consensus.
```
Comment (Walavouchey) [inline wiki/People/Beatmap_Nominators/General_Information/en.md:44]: i don't think "bn website" is a proper noun here

```suggestion
When a veto is sent to mediation, BNs and NAT members of the vetoed beatmap's mode will all be asked to submit individual votes on the issue. In the case of vetoes related to issues that are not mode-specific, all mode members will be asked to participate. The NAT will send you a message with additional information on how to vote using the [BN website](http://bn.mappersguild.com) when a veto mediation begins. At the end of mediation, all votes submitted will be compiled to determine a final upheld or dismissed consensus.
```

File: wiki/People/Beatmap_Nominators/Beatmap_Veto/en.md (hunk 1 above)
Comment (Walavouchey) [reply wiki/People/Beatmap_Nominators/Beatmap_Veto/en.md:29]: also

Comment (Walavouchey) [inline wiki/People/Beatmap_Nominators/Beatmap_Veto/en.md:29]: idk why this is a subheading. could rename the enclosing section to "veto mediation"

**Review by Hiviexd — APPROVED** [review]

**Pull request merged**